### PR TITLE
i18nize and refactor GlowstoneCommand

### PR DESCRIPTION
**Pull request by Pr0methean**
Subcommands are now looked up through an ImmutableSortedMap that ensures English case folding.

### CODE REVIEW

File: src/main/java/net/glowstone/command/glowstone/GlowstoneCommand.java [hunk 1]
```diff
@@ -30,162 +34,230 @@
...
-            // thread count
-            int threadCount = 0;
-            Set<Thread> threads = Thread.getAllStackTraces().keySet();
-            for (Thread thread : threads) {
-                if (thread.getThreadGroup() == Thread.currentThread().getThreadGroup()) {
-                    threadCount++;
+                // thread count
+                int threadCount = 0;
+                Set<Thread> threads = Thread.getAllStackTraces().keySet();
+                for (Thread thread : threads) {
```
Comment (aramperes) [inline src/main/java/net/glowstone/command/glowstone/GlowstoneCommand.java]: Could you refactor this to use a Stream?

File: src/main/java/net/glowstone/command/glowstone/GlowstoneCommand.java [hunk 2]
```diff
@@ -30,162 +34,230 @@
...
+        ABOUT("about") {
+            @Override
+            boolean execute(CommandSender sender, String label, String[] args,
+                    CommandMessages commandMessages) {
+                ResourceBundle b = commandMessages.getResourceBundle();
+                // some info about this Glowstone server
+                new LocalizedStringImpl("glowstone.about", b).send(sender);
+                LocalizedStringImpl t
+                        = new LocalizedStringImpl("glowstone.about._template", b);
+                sendBullet(sender, t, b, "glowstone.about.brand", Bukkit.getName());
```
Comment (aramperes) [inline src/main/java/net/glowstone/command/glowstone/GlowstoneCommand.java]: Could you rename the `b` and `t` variables to be more descriptive?

File: src/main/java/net/glowstone/command/glowstone/GlowstoneCommand.java [hunk 3]
```diff
@@ -200,37 +272,37 @@ private static void sendBullet(CommandSender sender,
...
+        switch (args.length) {
+            case 1:
+                return StringUtil.copyPartialMatches(args[0], SUBCOMMANDS,
+                        new ArrayList<>(SUBCOMMANDS.size()));
+            case 2:
+                switch (SUBCOMMAND_MAP.get(args[0])) {
+                    case PROPERTY:
+                        return StringUtil
+                                .copyPartialMatches(args[1],
+                                        System.getProperties().stringPropertyNames(),
```
Comment (aramperes) [inline src/main/java/net/glowstone/command/glowstone/GlowstoneCommand.java]: What's going on here with all the whitespace? It's hardly readable 😛

**Review by aramperes — DISMISSED** [review]

File: src/main/java/net/glowstone/command/glowstone/GlowstoneCommand.java (hunk 2 above)
Comment (Pr0methean) [reply src/main/java/net/glowstone/command/glowstone/GlowstoneCommand.java]: I'll look into how much longer they can be while still fitting most of the sendBullet calls on one line each.

**Review by aramperes — APPROVED** [review]

**Pull request merged**